.gitignore

```diff
@@ -10,5 +10,5 @@ static-collected
 static/.DS_Store
 
 .env
-
+/staticfiles
```

Comment:
Why are you ignoring this?

Reply:
@jimmykamau I did collectstatic to ensure the app runs locally, I did not want to push those static changes made in my branch.